automated-deployments-using-github-actions.md

```diff
@@ -0,0 +1,300 @@
+
+## Prerequisites 
+
+In order to follow along with this tutorial, it is expected that you satisfy the following requirements;
+
+
+- Have an [Okteto Cloud](https://okteto.com/) Account
+- Have the [Okteto CLI](https://okteto.com/docs/getting-started/installation/index.html) installed on a host machine
+- Have a [GitHub account](https://github.com/) with Git installed on a host machine
+- Have a basic understanding of the Python programming language
+- Have an installation of [Docker](https://www.docker.com/) on your local machine.
+
+
+## Step 1: Clone a Sample Python Flask App
+
+To get started, fork the sample application from it’s repository [here](https://github.com/vickywane/okteto-flask-app). After forking the repository, execute the command below from your terminal to clone your forked copy of the repository to your host machine;
+
+**Note**: *Replace the* `*GITHUB_USERNAME*` *in the URL below with your GitHub username to match the remote origin of the forked repository.* 
+
+
+    git clone https://github.com/{{GITHUB_USERNAME}}/okteto-flask-app
+
+ 
+Going through the `flaskr.py` file within the cloned `okteto-flask-app` folder in your preferred code editor, you would observe that it contains three API routes that are used for performing a create, retrieve and delete operation against a connected Couch database.
+
+The cloned project also contains a `Dockerfile` and `docker-compose.yml` file that contains all defined steps and services needed to create a multi-stage build for this application. 
+
+To run this application, execute the [docker-compose](https://docs.docker.com/compose/) command below to build and run the application container that consists of a `database` and `app` service.
+
+
+    docker-compose up --build
+
+To test the application above, execute the command below from a new terminal window to make a POST request to the `/api/customer` api route within the flask API using [cURL](https://curl.se/) which inserts a new document into the customer collection within the running Couch database through it’s RESTful Apiserver.
+
+
+    curl -X POST -d '{"name":"Victory Nwani","occupation":"Software Engineer"}'  -H 'Content-Type: application/json' http://localhost:5050/api/customer
+
+To view the data inserted from the `POST` request above, execute the command below from your terminal to make a GET request to the `/api/customer` api route within the flask API using [cURL](https://curl.se/) which retrieves all documents in the customer collection within the couch database.
+
+
+    curl http://localhost:5050/api/customer
+
+
+![](https://lh3.googleusercontent.com/1zojwJ4AEJ9siyHAOxv4Vexk6tK3-UsCW9CBB6KyZKihQUMebSi2xbk11VFPnBWvybnXXK3ZfgyjXD400w-Nzj2JNK3rA6Q08gx8bPQl75Igw7dVFQo7I7VHMPn0ml9j_7MmE7G_)
```

Comment:
Replace the links here with the name of the image from the zip file you sent. Also, add a text in the []. 


Apply this change uniformly.